ovip-0014.md

```diff
@@ -38,18 +38,25 @@ Initiator (Originator VASP)
 | Version     |                                      | `version`     | String       | 1..1  | fixed value `1.0`   |
 | Originator  | see [ovip-0013](https://github.com/OpenVASP/ovips/blob/master/ovip-0013.md#12-originator) (1.2) for elements | `originator`  |              | 1..1  |                     |
 | Beneficiary | see [ovip-0013](https://github.com/OpenVASP/ovips/blob/master/ovip-0013.md#13-beneficiary) (1.3) for elements | `beneficiary` |              | 1..1  |                     |
-| Transfer    |                                      | `transfer`    |              | 1..1  |                     |
-|             | Virtual asset type                   | `va`          | String       | 1..1  |                     |
-|             | Transfer type                        | `ttype`       | dt.TFR.ttype | 1..1  |                     |
-|             | Amount                               | `amount`      | Decimal      | 1..1  | 18 digits precision |
-| Comment     |                                      | `comment`     | String       | 0..1  |                     |
+| Transfer    |                                      | `transfer`    |                    | 1..1  |                     |
+|             | Virtual asset identifier             | `vaid`        | String             | 1..1  |                     |
+|             | Virtual asset identifier scheme      | `vaidscheme`  | dt.TFR.vaidscheme  | 1..1  |                     |
+|             | Transfer type                        | `ttype`       | dt.TFR.ttype       | 1..1  |                     |
+|             | Amount                               | `amount`      | Decimal            | 1..1  | 18 digits precision |
+| Comment     |                                      | `comment`     | String             | 0..1  |                     |
 
 **dt.TFR.ttype**
 
 | Code   | Description            |
 | :----- | :--------------------- |
 | `BCTX` | Blockchain transaction |
 
+**dt.TFR.vaidscheme**
+
+| Code   | Description            |
+| :----- | :--------------------- |
+| `ITIN` | International Token Identification Number (ITIN) |
```

Comment:
Looking one step ahead into the implementation, where can the tables with those id numbers be obtained?

Reply:
ITSA offers a public api to access information on all issued token identifiers

Reply:
@nbundi can we please document here on how to access that public API or reference documentation that explains this? This document's target audience are developers who are implementing OpenVASP. When they implement `ITIN` they are left in the dark with the current state of this document.

Reply:
We sure can add some reference to official ITIN docs/resources. We should not do it in the Transfer message definition or the OpenVASP protocol specs in general though as the location of such resources may change over time. Think of ITIN as the ISIN for virtual assets/tokens. Protocols making use of ISIN don't point to external resources giving more context on the ISIN itself.

Reply:
> We sure can add some reference to official ITIN docs/resources.

:+1:

> as the location of such resources may change over time.

I don't think that's a problem. This document can and will be edited as it's the case with this pull request.

> Protocols making use of ISIN don't point to external resources giving more context on the ISIN itself.

Perhaps that's because ISIN is an established and widely adopted standard while ITIN is aspiring to become one :wink: